docs/Modules/bpmn-modules/modeling.md

```diff
@@ -9,4 +9,160 @@ parent: "BPMN Modules"
 grand_parent: "Modules"
 ---
 
-# modeling
\ No newline at end of file
+# modeling
+
+This module activates the BPMN 2.0 modeling features [over the common modeling module](/modules/common-modules/modeling). It means that this module contains every API exported by the common modeling module.
+
+Additional APIs over the common modeling module are:
+
+* [updateLabel](#updatelabel)
+* [connect](#connect)
+* [updateProperties](#updateproperties)
+* [resizeLane](#resizelane)
+* [addLane](#addlane)
+* [splitLane](#splitlane)
+* [setColor](#setcolor)
+
+## updateLabel
+
+`updateLabel` API updates labels and changes the names of elements. This API fires
+[elements.changed](/events/common-events/elements-changed) event and executes [element.updateLabel](/commands/bpmn-commands/element-updateLabel) command.
+
+### Usage
+
+```javascript
+const elementRegistry = bpmnJS.get('elementRegistry');
+const modeling = bpmnJS.get('modeling');
+
+const startEvent_1 = elementRegistry.get('StartEvent_1');
```

Comment:
We should stick with our code style and use camelCase in favor of snake_case.

Reply:
Done 👍